src/pages/poe/characters/index.tsx

```diff
@@ -261,82 +290,170 @@ export default function Characters({
+                        </td>
+                        <td>
+                            <ul className="flex flex-row space-x-3 justify-left">
+                            <li className="text-center list-none w-1/5">
+                                {snapshot.level}
+                            </li>
+                            <li className="list-none">
+                                <StyledTooltip
+                                texts={[`${snapshot.characterClass}`]}
+                                placement="right"
+                                className="bg-slate-800"
+                                >
+                                <Image
+                                    src={`/assets/poe/classes/${snapshot.characterClass}.png`}
+                                    alt={snapshot.characterClass}
+                                    width={39}
+                                    height={30}
+                                />
+                                </StyledTooltip>
+                            </li>
+                            </ul>
+                        </td>
+
+                        <td>
+                            {snapshot.mainSkillKey ? (
+                            <li className="list-none">
+                                <StyledSkillImageTooltip
+                                texts={[`${snapshot.mainSkillKey}`]}
+                                placement="left"
+                                title="Skills"
+                                imageString={snapshot.mainSkillKey}
+                                className="bg-slate-800"
+                                >
+                                <Image
+                                    src={`/assets/poe/skill_icons/${snapshot.mainSkillKey}.png`}
+                                    alt=""
+                                    width={39}
+                                    height={30}
+                                />
+                                </StyledSkillImageTooltip>
+                            </li>
+                            ) : null}
+                        </td>
+
+                        <td className="font-semibold text-red-600">
+                            {snapshot.life}
+                        </td>
+                        <td className="font-semibold text-teal-300">
+                            {snapshot.energyShield}
+                        </td>
+                      </tr>
+                    ))}
+                </tbody>
+            </table>
+        </StyledCard>);
 }
 
+/**
+ * Props for {@link StyledMultiSearch}
+ */
+type StyledMultiSearchProps = { 
+    value: string, 
+    onValueChange: (e: any)=>void, 
+    onDateChange: (e: any)=>void 
+};
+
+/**
+ * Styled search input for filtering aggregates on the {@link Characters} page
+ */
+function StyledMultiSearch({ 
+    value, 
+    onValueChange, 
+    onDateChange }: StyledMultiSearchProps) {
+    return (
+        <StyledCard title={"Search"}>
```

Comment:
This and the `StyledAggregatorPanel` seem like they don't really do anything and also aren't really long in the first place.